include/roboteam_ai/analysis/play-utilities/invariants/AlwaysFalseInvariant.h

```diff
@@ -22,7 +22,7 @@ class AlwaysFalseInvariant {
      * @param field the current field state
      * @return true if the ball belongs to us, false otherwise
      */
-    static bool isValid(rtt::ai::world::World *world, const Field *field);
+    static bool isValid(rtt::ai::world::World *world, const Field &field);
```

Comment:
Why do you make this function static?

Reply:
Because it needs to be called from outside its class